src/core/patterns/PatternsProvider.tsx

```diff
@@ -7,10 +7,11 @@ namespace PatternsProvider {
     patterns: Pattern[];
     loading: boolean;
     error: string;
-    getPatterns?(): void;
+    getPatterns?(query?: string): void;
   }
 
   export interface Props {
+    getOnInit?: boolean;
```

Comment:
getOnInit should be after children. We have convention - first normal parameters in aflphabetic order and after all nullable parameters also in alphabetic.